github-pullrequest-plugin/src/main/java/com/github/kostyasha/github/integration/branch/events/impl/commitchecks/GitHubBranchCommitCheck.java

```diff
@@ -30,4 +31,16 @@ public GitHubBranchCommitCheckDescriptor getDescriptor() {
      * @return <code>GitHubBranchCause</code> instance indicating if the build should be skipped, <code>null</code> otherwise.
```

Comment:
So i missed it... Bad that it differs from common logic because event logic solves everything